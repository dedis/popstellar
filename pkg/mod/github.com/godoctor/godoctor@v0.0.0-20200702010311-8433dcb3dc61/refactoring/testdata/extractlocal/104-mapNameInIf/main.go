package main

import "fmt"

func main() {
	m := map[int]int{1: 11, 2: 22}
	if v, found := m[2]; found { // <<<<< var,7,17,7,17,newVar,pass
		fmt.Println(v)
	}
}
