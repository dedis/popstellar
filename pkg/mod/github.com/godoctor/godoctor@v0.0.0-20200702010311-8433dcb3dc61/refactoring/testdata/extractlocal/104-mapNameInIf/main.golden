package main

import "fmt"

func main() {
	m := map[int]int{1: 11, 2: 22}
	newVar := m
	if v, found := newVar[2]; found { // <<<<< var,7,17,7,17,newVar,pass
		fmt.Println(v)
	}
}
